src/systems/interactions.js

```diff
@@ -170,83 +203,41 @@ AFRAME.registerSystem("interaction", {
       );
       if (state.hovered) {
         const entity = state.hovered;
-        const isHoldable = entity.components.tags && entity.components.tags.data.isHoldable;
         const isFrozen = this.el.is("frozen");
         const isPinned = entity.components.pinnable && entity.components.pinnable.data.pinned;
-        if (isHoldable && userinput.get(options.grabPath) && (isFrozen || !isPinned) && canMove(entity)) {
+        if (
+          isTagged(entity, "isHoldable") &&
+          userinput.get(options.grabPath) &&
+          (isFrozen || !isPinned) &&
+          canMove(entity)
+        ) {
           state.held = entity;
         }
       }
     }
   },
 
-  tick2(sfx) {
-    if (!this.el.is("entered")) return;
+  tick2() {
```

Comment:
we did not need to and should change userinput and interaction to be a hubs-system like the others. I did not do it at the time so that the logic-changing commits weren't mixed with a big renaming commit but I haven't gotten around to doing the renaming.